engine-tests/src/test_utils/mod.rs

```diff
@@ -406,6 +407,11 @@ impl AuroraRunner {
             Ok(bytes)
         }
     }
+
+    pub fn with_random_seed(mut self, random_seed: Vec<u8>) -> Self {
```

Comment:
I think it would be nicer if this took `[u8; 32]` and turned it into a `Vec` when passing to the context. The seed is supposed to have a fixed size.

Reply:
Refactored everything to use `H256`.